providers/azure/aci.go

```diff
@@ -216,6 +217,7 @@ func NewACIProvider(config string, rm *manager.ResourceManager, nodeName, operat
 	p.nodeName = nodeName
 	p.internalIP = internalIP
 	p.daemonEndpointPort = daemonEndpointPort
+	p.startTime = time.Now()
```

Comment:
Cheating here a bit... really was just checking if this was an issue with metrics-server... doesn't appear to be.